druid-shell/src/dialog.rs

```diff
@@ -21,7 +21,16 @@ use std::path::{Path, PathBuf};
 /// This path might point to a file or a directory.
 #[derive(Debug, Clone)]
 pub struct FileInfo {
+    /// The path to the selected file.
+    /// On macOS, this is already rewritten to use the extension that the user selected
+    /// with the `file format` property.
```

Comment:
```suggestion
    /// The path to the selected file.
    ///
    /// On macOS, this is already rewritten to use the extension that the user selected
    /// with the `file format` property.
```

Kind of like git commit messages, rust doc comments treat the first line as a 'summary', and then want a newline between it and the rest of the docs. This is displayed inline beside an item when the docs are rendered. 👌